request_a_govuk_domain/request/models/application.py

```diff
@@ -23,6 +24,9 @@ class Application(models.Model):
 
     id = models.BigAutoField(primary_key=True)
     reference = models.CharField(max_length=REF_NUM_LENGTH, null=False)
+    owner = models.ForeignKey(User, on_delete=models.CASCADE, null=True)
```

Comment:
Regarding on_delete, Do we want to delete the application when User is deleted? Would SET_NULL be a better option, considering the field is nullable?